### DISCUSSION TITLE
Is possible to get Analytics alarm from axis camera using windows SDK

### DISCUSSION THREAD

**Discussion by schackovidsys** [AxisCommunications/discussions#207] [Axis Windows SDKs] [answered]
Hi,

We are using you SDK to integrate the Axis camera with PSIM product. We are getting the live video, 
However, we would also like to receive the analytics alarm from axis camera.
How to get analytics alarms using windows SDK?

Thanks.

**Comment by anders-axis** [comment, marked as answer]
You can use AXIS Media Parser SDK to retrieve any device events, including analytics events, over RTSP. See sample code: "Visual C#\EventNotification".

**Reply by anders-axis** [reply]
Tip: [AXIS Metadata Monitor](https://www.axis.com/developer-community/axis-metadata-monitor) is a good tool for exploring what kind of events is sent from a device and how they look like.
